editor/src/messages/portfolio/document/node_graph/node_graph_message_handler.rs

```diff
@@ -121,6 +121,7 @@ pub struct NodeGraphMessageHandler {
 	pub layer_path: Option<Vec<LayerId>>,
 	pub network: Vec<NodeId>,
 	has_selection: bool,
+	graph_open: bool,
```

Comment:
Don't we already have a boolean in the Rust code storing if the graph overlay is open?

Reply:
We cannot access the portfolio message manager from within the graph message handler whilst creating the actions. This causes the node graph delete shortcut to override other delete actions from the path tool.

Reply:
I am unsure why the graph open boolean got moved from the node graph message handler in the first place as we now have duplicate state.

Reply:
I'm pretty sure it was always in the portfolio message handler since back when I originally wrote it.

From a user perspective, it's basically a question a whether it's more useful/intuitive for the graph overlay state to follow the active document tab or be global to the application. It's the same question as the active tool and the working colors, both of which are global. For consistency with those, that's why I also picked to have the overlay be global; however I don't necessarily feel strongly about keeping it that way. If it's a significant code simplification to have it follow the document, I'm okay with that change. But we should only keep the state in a single boolean in the code. Would you mind refactoring it to remove the redundancy?